gpytorch/inference/posterior_models/variational_gp_posterior.py

```diff
@@ -96,6 +96,7 @@ def marginal_log_likelihood(self, output, train_y, num_samples=5):
         epsilon = Variable(torch.randn(num_inducing, num_samples))
         samples = chol_var_covar.mm(epsilon)
         samples = samples + self.variational_mean.unsqueeze(1).expand_as(samples)
+

```

Comment:
Haha it's best to avoid spurious changes like this. Causes unnecessary conflicts.